src/etc/man/cargo-test.1

```diff
@@ -1,66 +1,63 @@
 '\" t
 .\"     Title: cargo-test
-.\"    Author: [see the "AUTHOR(S)" section]
-.\" Generator: Asciidoctor 2.0.10
-.\"      Date: 2020-06-25
+.\"    Author: [see the "AUTHORS" section]
+.\" Generator: Asciidoctor 1.5.5
+.\"      Date: 2020-07-28
```

Comment:
The manpage got a lot of weird little changes like this, was this caused by my Asciidoctor version? This was just the default version that I installed from `apt`, so I'm not sure if that's fine or not. Plus there's a ton of stuff replacing `\fB` with `\f[CR]` (I assume some kind of "line break -> carriage return" thing).

Reply:
Not like the date stuff, that makes sense